test/setup

```diff
@@ -6,20 +6,22 @@ git config core.autocrlf input
 set -e
 
 # Set the GIT_SUBREPO_ROOT for testing.
-source $PWD/.rc
+source "$PWD"/.rc
+
+# Get the location of this script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Bash doesn't need quotes on assignments. These are the same:
```
x="$( ... )"
x=$( ... )
```
The `$( ... )` is parsed as a single expression even though it has ws in it. 

Please remove the quotes. 

At some point after the release, I'd like to go over all the Bash stuff, to bring it up to a set of standards that I've developed since starting subrepo.